newIDE/app/src/UI/Toolbar.js

```diff
@@ -22,7 +22,7 @@ export const Toolbar = React.memo<ToolbarProps>(
           backgroundColor: gdevelopTheme.toolbar.backgroundColor,
           flexShrink: 0,
           display: 'flex',
-          overflowX: 'auto',
+          overflowX: 'overlay',
```

Comment:
what does this fix?

Reply:
I took the liberty of putting the scrollbar "inside" the div of the toolbar, much like what is done for the tabs. This only works on Chrome, but allows to avoid a different height when you go between editors and some need a scrollbar and some don't.
On Firefox, we hide it. On iOS, scrollbars are hidden and "inside" the divs by default.